codecs/h264_spreader.go

```diff
@@ -0,0 +1,401 @@
+/*
```

Comment:
Not sure where this struct/file should go ? Top folder ? A new internal package ?